feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDetail.kt

```diff
@@ -136,6 +136,65 @@ fun SessionDetailTopAppBar(
     )
 }
 
+@Composable
+fun SessionDetailBottomAppBar(
```

Comment:
Thanks! In this project, we put the children's compose function below the parent composable function. So can you move it below SessionDetailScreen? 🙏

Reply:
Thanks for your review.
I've corrected!